README.md

```diff
@@ -2,6 +2,18 @@
 
 A modal dialog opened with a &lt;details&gt; button.
 
+## DEPRECIATED WARNING
+
+This web component has been depreciated. There are a number of accessibility concerns with this approach and so we no longer recommend using this component.
```

Comment:
```suggestion
## DEPRECATION WARNING

This web component has been deprecated. There are a number of accessibility concerns with this approach and so we no longer recommend using this component.
```

I think `deprecated` is more used that `depreciated`?

Reply:
Thanks @manuelpuyol - I'm not sure why I always want to put `depreciated`, a appreciate the correction 😅